atlasdb-cassandra/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I'm always scared when I see this; it often indicates that we're not necessarily doing the right thing (even though I am most definitely guilty of this in the past!)

Reply:
Ha, I saw this pretty much everywhere except this particular package.

This was needed to mock CqlCluster. I debated whether it was worth refactoring CqlCluster into an interface and an implementation (and in fact, having a noncloseable version so that consumers don't just close a cqlcluster managed by the container), but was not entirely convinced it was the best thing to do in this PR.

If it's better to go down that route, I'll make a separate PR to do that refactor.

Reply:
Just because it exists everywhere does not mean it's right!!! However, yes, you will find that in AtlasDB we do mock final classes frequently. This comment was around whether we want to extract out a generic interface or not.